raw_description/urdf/base_mini/base.urdf.xacro

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/base_mini/base.gazebo.xacro" />
+
+  <!-- drive_wheel module -->
+  <xacro:include filename="$(find raw_description)/urdf/mecanum_wheel/mecanum_wheel.urdf.xacro" />
+
+
+  <!-- properties -->
+  <xacro:property name="caster_offset_x" value="0.115" />
+  <xacro:property name="caster_offset_y" value="0.09" />
+  <xacro:property name="caster_offset_z" value="0.00" />
+
+  <xacro:property name="battery_offset_x" value="0.0" />
+  <xacro:property name="battery_offset_y" value="-0.008" />
+  <xacro:property name="battery_offset_z" value="0.065" />
+
+  <xacro:property name="base_mass" value="43.1" />
+  <xacro:property name="battery_mass" value="40" />
```

Comment:
are these values (mass and dimensions) correct?

Reply:
no (fixed)